src/components/MainParticipantInfo/MainParticipantInfo.tsx

```diff
@@ -104,8 +104,26 @@ const useStyles = makeStyles((theme: Theme) => ({
       top: 0,
     },
   },
+  switchedOffMessage: {
+    opacity: 0,
+    animationName: '$showMessage',
+    animationDuration: '0.5s',
+    animationDelay: '2s',
+    animationFillMode: 'forwards',
+    color: 'white',
+  },
+  '@keyframes showMessage': {
+    '0%': {
+      opacity: 0,
+      visibility: 'hidden',
+    },
+    '100%': {
+      opacity: 1,
+      visibility: 'visible',
+    },
```

Comment:
With something like this, we can give the message a nice fade in:
```
'@keyframes showMessage': {
      '0%': {
        opacity: 0,
        visibility: 'hidden',
      },
      '80%': {
        opacity: 1,
        visibility: 'visible',
      },
      '100%': {
        opacity: 1
      },
    }
```

No need to use these exact values. Play with it and see what works.

Reply:
I like the idea of fading the message in, but I'm not sure I'm seeing much of a difference in the UI using your example. I tried changing `opacity` to something like `0.8` for `80%` and that looked kind of glitchy. I did notice that increasing the `animationDuration` from `0.5s` to `1s` created a fade in effect. I'm wondering if you're seeing something that I'm not seeing though, so you might need to show me?

Reply:
Hmm, I wasn't originally seeing a fade in, but now I am. Not sure what changed, but it looks good now :+1: